### DISCUSSION TITLE
Guida Openhab 3

### DISCUSSION THREAD

**Discussion by Muplex** [indomus/forum#222] [HUB: openHAB]
Ciao,
sono un utente di Openhab da oltre 6 anni.
Dall'uscita della versione 3 credo che il sistema sia abbastanza maturo e strutturalmente solido a livello software da poter iniziare a farci una guida che resterà valida per molto tempo.

Vorrei che InDomus avesse anche una sezione dedicata a Openhab oltre che a HomeAssistant. C'è qualcuno a cui piacerebbe collaborare per la stesura di alcuni articoli? 
Al momento è un idea, non so bene da dove inizire....Qualche moderatore, potrebbe indicarmi qualche lettura mirata per iniziare la stesura di un articolo pubblicabile ?

Grazie e saluti,
Muplex

**Comment by indomus** [comment]
Ciao, se ti senti di collaborare sei il benvenuto. La sezione su inDomus c'è, ma è piuttosto scarna dato il poco seguito che c'è. Contattaci su crew at indomus.it e ne parliamo volentieri.

**Reply by Muplex** [reply]
fatto =)
